build.cake

```diff
@@ -141,23 +123,24 @@ Task("Externals-Apple")
     .WithCriteria(() => IsRunningOnUnix())
     .Does(() =>
 {
-    var zipFile = System.IO.Path.Combine(AppleExternals, "apple.zip");
-    var zipXCFrameworkFile = System.IO.Path.Combine(AppleExternals, "apple-xcframework.zip");
     var XCFrameworkOutputDir = System.IO.Path.Combine(AppleExternals, "xcframework");
-
-    CleanDirectory(AppleExternals);
-
-    // Download framework and xcframework files.
-    var authParams = Argument("StorageAuthParams", EnvironmentVariable("STORAGE_AUTH_PARAMS"));
-    using (VerboseVerbosity())
+    var appleExternalsPath = System.IO.Path.Combine(AppleExternals, "*");
```

Comment:
How do we get artifacts locally now? We would need some documentation on the process.

Reply:
Should we write like a comment inside the cake script when running locally to download artifacts from build pipelines and place them in externals directory?